packetbeat/protos/smtp/smtp_test.go

```diff
@@ -0,0 +1,196 @@
+// +build !integration
+
+package smtp
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please run `make fmt` to automatically reorder the imports using `goimports` (we group by stdlib, third-party, then elastic).